### PR TITLE
error message fix

### PR DESCRIPTION
**Pull request by zhangtong007**

### CODE REVIEW

File: pkg/dal/storage-client/data_access/mysql.go [hunk 1]
```diff
@@ -460,7 +460,7 @@ func (s *StorageSqlImpl) QuerySloTraceDataWithPodUID(data interface{}, podUid st
 func (s *StorageSqlImpl) QueryCreateSloWithResult(data interface{}, requestParams *model.SloOptions) error {
 
 	if requestParams == nil || requestParams.Result == "" {
-		return fmt.Errorf("the params is error")
+		return fmt.Errorf("the params is error, delieryresult needed")
```
Comment (linuzb) [inline pkg/dal/storage-client/data_access/mysql.go:463]: Is it better to define the error code as a constant?

Comment (zhangtong007) [reply pkg/dal/storage-client/data_access/mysql.go]: I provide a custom error interface. we can classify errors by category, and define them globally, and return them uniformly through the interface.

Comment (zhangtong007) [reply pkg/dal/storage-client/data_access/mysql.go]: if we need to add a new error，such as NoPodUID，we just need to add it as shown below
test

**Review by linuzb — APPROVED** [review]
LGTM

**Review by d3c3mber — APPROVED** [review]
lgtm

**Pull request merged**